src/inc/quic_platform_posix.h

```diff
@@ -688,7 +688,11 @@ CxPlatEventInitialize(
     _In_ BOOLEAN InitialState
     )
 {
-    pthread_condattr_t Attr = {0};
+#if defined(CX_PLATFORM_DARWIN)
+    pthread_condattr_t Attr = {0, 0};
+#else
+    pthread_condattr_t Attr = {{0, 0}};
+#endif
```

Comment:
Easier to just do a `CxPlatZeroMemory(&Attr, sizeof(Attr))`

Reply:
Ditto for the other cases too.

Reply:
updated. For reference is was failing with 
```
Users/furt/github/wfurt-msquic/src/inc/quic_platform_posix.h:692:36: error: excess elements in scalar initializer [-Werror]
    pthread_condattr_t Attr = {{0, 0}};
                                   ^
1 error generated.
```